doc/reference_en.html

```diff
@@ -2022,6 +2022,25 @@ <h3 class="not-for-contents">1. Why not to use systems like map-reduce?</h3>
 If you have an ORDER BY clause with some small LIMIT after a GROUP BY, then ORDER BY will not consume significant amount of memory. But if no LIMIT is provided, don't forget to turn on external sorting (%%max_bytes_before_external_sort%%).
 
 
+<h5>LIMIT N BY modifier</h5>
+
+LIMIT %%N%% BY %%COLUMN%% allows you to restrict top %%N%% rows per each group of %%COLUMN%%
+
+Example:
+
+%%SELECT
+    domainWithoutWWW(URL) AS domain,
+    domainWithoutWWW(REFERRER_URL) AS referrer,
+    count() cnt
+FROM hits
+GROUP BY domain, referrer
+ORDER BY cnt DESC
+LIMIT 5 BY domain
```

Comment:
No mention about LIMIT BY is unrelated to LIMIT and query could even contain both clauses (ordinary LIMIT could go after LIMIT BY).